content/archive/ARCHIVED; Open Sourcing Tutor-GPT.md

```
@@ -73,9 +80,7 @@ Notice how Bloom reasons it should indulge the topic, validate the student, and
 Aside from these edgier cases, Bloom shines helping students understand difficult passages (from syntactic to conceptual levels) and giving writing feedback (especially competent at thesis construction). [Take it for a spin](https://discord.gg/udtxycbh).
 
 Ultimately, we hope [open-sourcing Bloom](https://github.com/plastic-labs/tutor-gpt#readme) will allow anyone to run with these elicitations and prompt to expand utility and support multiple domains. We’ll be doing work here too.
-
-## Bloom & Agentic AI
-
+# Bloom & Agentic AI
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)